inference-engine/tests/functional/plugin/myriad/single_layer_tests/detection_output.cpp

```diff
@@ -0,0 +1,550 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
why this is not a common test for all plugins?

Reply:
@ilya-lavrenov In fact, the OV already has a [test ](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/shared_test_classes/src/single_layer/detection_output.cpp) for DetectionOutput. But Myriad fails that test. Since we do the calculations in FP16, there are at least two reasons:

- Random data [generation](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/shared_test_classes/src/single_layer/detection_output.cpp#L90)
- Element-wise comparison function

However, unfortunately, our deprecated test also depends on the data. If you change the seed for random, it fails. And new tests fails on Windows. I have already created a ticket: #-48015

Reply:
@nikita-kud It is a bad practice to instantiate plugin specific test without instantiation from shared tests

Reply:
@iefode Is it a good practice to migrate deprecated tests to a new infrastructure?

Reply:
This one - yes :)

Reply:
In this case, it's not a bad refactoring (if I win over Windows)